app/javascript/gobierto_data/webapp/components/commons/VisualizationsSlot.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="gobierto-data-visualization--card">
+    <div class="gobierto-data-visualization--aspect-ratio-16-9">
+      <div class="gobierto-data-visualization--content">
+        <h4 class="gobierto-data-visualization--title">
+          {{ name }}
+        </h4>
+        <slot name="visualization" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "VisualizationSlot",
+  props: {
+    name: {
+      type: String,
+      default: ''
+    }
+  }
+}
+</script>
```

Comment:
If you've only one slot, you don't need no set a name, just `<slot />`.

In this case, you don't need to pass a prop neither, just use another slot, easy-peasy:

```suggestion
<template>
  <div class="gobierto-data-visualization--card">
    <div class="gobierto-data-visualization--aspect-ratio-16-9">
      <div class="gobierto-data-visualization--content">
        <h4 class="gobierto-data-visualization--title">
          <slot name="title" />
        </h4>
        <slot />
      </div>
    </div>
  </div>
</template>

<script>
export default {
  name: "VisualizationSlot",
}
</script>
```

Reply:
Cool, thanks